webapp/channels/src/components/common/hooks/scroll.ts

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+export function useScrollOnRender() {
```

Comment:
can you include a comment on what this function is and whats its purpose

Reply:
Done.